addons/base_automation/views/base_automation_views.xml

```diff
@@ -233,44 +229,41 @@
-                                        <div class="d-flex flex-column">
-                                            <field name="name" class="fs-2 fw-bold"/>
-                                            <field name="model_id" invisible="context.get('default_model_id')"/>
-                                            <div class="d-flex align-items-center gap-1" invisible="trigger in ['on_time', 'on_time_created', 'on_time_updated']">
-                                                <field name="trigger" />
-                                                <field name="on_change_field_ids" invisible="trigger != 'on_change'" class="my-1" />
-                                                <field name="trg_selection_field_id" invisible="trigger not in ['on_state_set', 'on_priority_set']" class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
-                                                <field name="trg_field_ref" invisible="trigger not in ['on_stage_set', 'on_tag_set']" class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
-                                                <field name="trigger_field_ids" invisible="trigger not in ['on_create_or_write']" class="my-1" />
-                                            </div>
-                                            <div class="d-flex align-items-center gap-1" invisible="trigger != 'on_time'">
-                                                <field name="trg_date_range"/>
-                                                <field name="trg_date_range_type" class="text-lowercase" />
-                                                <span class="flex-shrink-0">
-                                                    based on <field name="trg_date_id" invisible="trigger != 'on_time'"  class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
-                                                </span>
-                                            </div>
-                                            <div class="d-flex align-items-center gap-1" invisible="trigger not in ['on_time_created', 'on_time_updated']">
-                                                <field name="trg_date_range"/>
-                                                <field name="trg_date_range_type" class="text-lowercase" />
-                                                <field name="trigger" class="text-lowercase" />
-                                                <field name="trg_date_id" invisible="trigger != 'on_time'"  class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
-                                            </div>
-                                            <field name="trg_date_calendar_id" invisible="not trg_date_id or trg_date_range_type != 'day'" />
+                        <t t-name="kanban-card">
+                            <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active" />
+                            <div class="d-flex flex-column flex-md-row gap-3 flex-grow-1">
```

Comment:
the article is already d-flex flex-column, so this div is probably not necessary (you can set a class attribute on the t t-name="kanban-card" node, those classnames will be put on the article node